src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -105,6 +356,10 @@ function create_cache(mesh::VertexMappedMesh, equations,
     Pf = droptol!(sparse(Pf'), 100 * eps(eltype(Pf)))'
   end
 
+  # TODO: DGMulti; this is temporary, remove this once things are stable.
+  interp_matrix_gauss_to_face = TensorProductGaussFaceOperator(Interpolation(), dg)
+  Pf = TensorProductGaussFaceOperator(Projection(), dg)
```

Comment:
This should be resolved by removing the outdated comment as well as removing or commenting out the construction of obsolete versions of these operators.

Reply:
Weird, apparently that change wasn't committed b/c it got stashed. Just committed it now.